src/pycba/bridge.py

```diff
@@ -489,13 +489,14 @@ def plot_envelopes(self, env: Envelopes):
         subfigs = fig.subfigures(1, 2, wspace=0.07)
 
         # Shear and moment in left panel
-        subfigs[0].suptitle("Stress Resultants")
+        subfigs[0].suptitle("Force Resultants")
```

Comment:
Can you leave as "Stress Resultants" please - [correct terminology](https://en.wikipedia.org/wiki/Stress_resultants); i.e. shear and moment are integrals of the stresses over the cross section.

Reply:
You're right, I'll leave as it is! 
Thank you.